2/contrib/openshift/configuration/proxy.yaml.tpl

```diff
@@ -0,0 +1,8 @@
+jenkins:
+  proxy:
+    name: "${PROXY_HOST}"
+    port: ${PROXY_PORT}
+    userName: "${PROXY_USER}"
+    secretPassword: "${PROXY_PASSWORD}"
+    noProxyHost: "${NO_PROXY}"
+    testUrl: "http://redhat.com"
```

Comment:
if the parameter is not mandatory, maybe we should remove it, or have also a way to set it; because it may cause issues with disconnected environment or with proxies that does filter this url or any test url that we would specify.

Reply:
fixed